### PR TITLE
feat: complete persona UI gaps — spawn dialog, auto-fill, badge, tests

### PR DESCRIPTION
**Pull request by PrinceUmbrella**
## Summary

Completes the remaining persona UI gaps from #173, #174, and #175. Adds persona selection to the spawn dialog, auto-fill on selection, persona badges on tasks, CLI help examples, and edge case tests.

## Changes

**Spawn Dialog (#173)** — Persona dropdown added to new_chat spawn form. Selecting a persona auto-fills the runtime dropdown. WS bridge spawn handler now resolves persona (system prompt, runtime, model, maxTurns).

**Auto-fill + Override (#173)** — When a persona is selected in either task creation or spawn, the runtime field auto-populates from the persona config. The user can still change it (persona is a starting template, not a lock).

**Persona Badge (#173)** — Tasks with a persona assigned now show the persona name as a badge in the sidebar task tree, next to the dependency badge.

**CLI Examples (#172)** — Added `addHelpText` examples to `persona list` and `persona create` commands.

**Edge Case Tests (#175)** — Added tests for deleting a non-existent persona (no-op) and enforcing unique name constraint on update (throws).

## Screenshots

Captured against this branch and attached here to cover every visible change in the PR:

1. **Spawn dialog persona selector**
Spawn dialog persona selector

2. **Runtime auto-fill from persona selection**
Runtime auto-fill from persona selection

3. **Persona badges in the sidebar task tree**
Persona badges in the sidebar task tree

## Testing

- 133 server unit tests pass (including 11 persona-store tests)
- Build passes all packages
- CLI `persona create --help` shows examples
- Playwright used to capture the spawn-dialog auto-fill and sidebar badge flows from the live UI

Addresses remaining gaps from #173, #174, #175

### CODE REVIEW

File: packages/web/src/components/layout/UnifiedBar.tsx [hunk 1]
```diff
@@ -53,6 +53,29 @@ export function UnifiedBar({ viewMode, setViewMode }: Props): JSX.Element {
...
   const [taskPersonaId, setTaskPersonaId] = useState("");
+  const [spawnPersonaId, setSpawnPersonaId] = useState("");
+
+  /** When a persona is selected in the new_task form, auto-fill runtime if the persona specifies one. */
+  const handleTaskPersonaChange = (personaId: string): void => {
+    setTaskPersonaId(personaId);
+    if (personaId) {
+      const p = personas.find((x) => x.id === personaId);
+      if (p?.runtime) {
+        setTaskEnvId((prev) => prev); // keep env
```
[the remark this answers is not in the document]
Comment (PrinceUmbrella) [reply packages/web/src/components/layout/UnifiedBar.tsx]: Added PR screenshots for the UI changes in this branch: one showing persona selection auto-filling the runtime in the spawn dialog, and one showing the persona badge rendered in the sidebar task tree.

**Review by nick-pape — APPROVED** [review]
